client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/transport/netty/EPollAvailable.java

```diff
@@ -0,0 +1,34 @@
+package org.infinispan.client.hotrod.impl.transport.netty;
+
+import org.infinispan.client.hotrod.logging.Log;
+import org.infinispan.commons.logging.LogFactory;
+
+import io.netty.channel.epoll.Epoll;
+
+// This is a separate class for better replacement within Quarkus as it doesn't support native EPoll
+final class EPollAvailable {
+   static private final Log log = LogFactory.getLog(EPollAvailable.class, Log.class);
+   static final boolean USE_NATIVE_EPOLL = useNativeEpoll();
+
+   private static final String USE_EPOLL_PROPERTY = "infinispan.server.channel.epoll";
+   private static final boolean IS_LINUX = System.getProperty("os.name").toLowerCase().startsWith("linux");
+   private static final boolean EPOLL_DISABLED = System.getProperty(USE_EPOLL_PROPERTY, "true").equalsIgnoreCase("false");
```

Comment:
This looks exactly like ```Boolean.getBoolean``` :)

Reply:
This is just what the code was before, but I agree :)